webapp/store/user/reducer.js

```diff
@@ -1,9 +1,9 @@
 import { exportReducer } from '@webapp/utils/reduxUtils'
 
-import * as UserState from '@webapp/user/userState'
+import * as UserState from '@webapp/store/user/state'
```

Comment:
```import { UserState } from '@webapp/store/user'```

Reply:
This gives cycle error, see commit [1bdb0fa](https://github.com/openforis/fra-platform/pull/606/commits/1bdb0fa5f2572b7ee004ffb6dacc898334cf160b)